distributionviewer/core/static/js/app/router.js

```diff
@@ -12,7 +12,7 @@ export default (
   <Router history={browserHistory}>
     <Route component={MainLayout}>
       <Route path="/" component={Home} />
-      <Route path="/chart/:chartId" component={ChartDetailContainer} />
+      <Route path="/chart/:chartName" component={ChartDetailContainer} />
```

Comment:
This will give us paths like */chart/architecture/*. Not sure if *chart* is the best name here. *metric*? *graph*? Open to suggestions.